assets/base/i18n/en.json

```diff
@@ -959,6 +959,8 @@
   "calling_screen": "Calling",
   "center_panel.recent": "Click here to jump to recent messages. ",
   "channel_header.addMembers": "Add Members",
+  "channelHeader.addToFavorites": "Add to Favorites",
+  "channelHeader.removeFromFavorites": "Remove from Favorites",
```

Comment:
Something else we should figure out soon is how the localization strings will get synced between the repos